pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/EventService.java

```diff
@@ -96,8 +98,11 @@ protected static boolean triggerDssByKitEvent(KitDDPNotification kitDDPNotificat
     @VisibleForTesting
     protected static boolean triggerDssWithEvent(@NonNull String eventType, DDPInstance ddpInstance, long eventDate,
                                         @NotNull String ddpParticipantId, @NotNull String eventInfo, KitReasonType reason) {
-        final long initialInterval = 500; // base delay in milliseconds
-        final double multiplier = 2.0; // exponential backoff multiplier
+
+        final long initialInterval = DSMConfig.getIntFromConfig(ApplicationConfigConstants.EVENT_RETRY_INTERVAL_MS); // base delay in
```

Comment:
I think it would be better to pass these values in as args to `triggerDssWithEvent()`, instead of reading the config values inside the method.  Reading global config values inside a method like this tends to make testing harder.  They could also become instance variables in `EventService`, and then the methods that need them could move to instance methods instead of static methods.  Alternatively, they can become static final variables at the class level.

Reply:
It was cleaner to make them static final variables in the class level so I moved them there